### DISCUSSION TITLE
记录对er2260t路由器的一些配置 https://blog.03k.org/post/er2260t_setting.html

### DISCUSSION THREAD

**Discussion by kkkgo** [kkkgo/blog.03k.org#24] [blog]
https://blog.03k.org/post/er2260t_setting.html

**Comment by cvn77** [comment]
请问楼主，按照你这个修改可以设置本地ipv6地址作为dns地址吗？

**Reply by kkkgo** [reply]
可以禁止下发IPV6的DNS地址，然后用IPv4的就行。毕竟用IPv4的DNS不影响你解析IPV6 AAAA记录。

**Comment by hsj1992** [comment]
想请问现在用的主路由是哪一款？想参考一下换掉ER2260T。
（刷了新版官方固件但仍然会有PPPOE重拨时LAN口失效的情况而且SSH密码也变动无法登陆了....）

**Reply by kkkgo** [reply]
现在用的xdr6088刷openwrt，图他的联发科830 CPU性能够强刷机自由。找个带硬件加速的固件。
不过是双2.5G。双10G的可以看威联通的那几款路由器，还有出厂就带openwrt的。就是有点小贵。

**Reply by WalterDSU** [reply]
最近京东云雅典娜也支持了OP了，性能上会不会雅典娜可以替代xdr6088？还有，ER2260T有了新的固件，不知道是否解决了上面提及的问题？

**Comment by Lovest20018** [comment]
r5408m可以修改。但一堆的莫名其妙问题。不过头疼的还是shield tv 只要休眠后ipv6就无法连接。只能开关下网络才可以。算了。还是关闭ipv6安逸。比较现在ipv6使用场景就直播源。
